tests/utils/test_env_pack.py

```diff
@@ -247,6 +249,43 @@ def test_pack_env_for_databricks_model_serving_runtime_version_check(tmp_path, m
             assert Path(artifacts_dir).exists()
 
 
+def test_validate_env_pack_with_valid_inputs():
+    # valid string should not raise; None should be treated as no-op
+    assert _validate_env_pack("databricks_model_serving") is None
+    assert _validate_env_pack(None) is None
+    cfg = EnvPackConfig(name="databricks_model_serving", install_dependencies=True)
+    assert _validate_env_pack(cfg) is None
+
+
+def test_validate_env_pack_with_invalid_string_raises():
+    with pytest.raises(MlflowException.invalid_parameter_value):
```

Comment:
```suggestion
    with pytest.raises(MlflowException, match="..."):
```

let's specify match.